tests/Feature/Subscription.php

```diff
@@ -464,7 +464,8 @@ public function testUpdateSubscriptionPlanToPlanWithDifferentBillingCycleFromYea
         $this->assertTrue($subscription->is(Status::canceled()));
 
         $rollover = $subscription->getRemainingValue()->getAmount();
-        $this->assertEquals($rollover, 1726.83);
+        $this->assertGreaterThan(1726.80, $rollover);
+        $this->assertLessThan(1726.86, $rollover);
```

Comment:
Can we update this to leverage:
https://docs.phpunit.de/en/8.5/assertions.html#assertequalswithdelta

```php 
$this->asserEqualsWithDelta(1726.83, $rollover, 0.3)
```